components/formats-api/src/loci/formats/readers.txt

```diff
@@ -109,6 +109,7 @@ loci.formats.in.XLEFReader            # xlef
 loci.formats.in.OlympusTileReader     # omp2info
 loci.formats.in.DCIMGReader           # dcimg
 loci.formats.in.JDCEReader            # jdce
+loci.formats.in.TissueFAXSReader      # aqproj
```

Comment:
This could probably be update to `aqproj, tfcyto` since both extensions will be supported